nautobot/core/jobs/__init__.py

```diff
@@ -0,0 +1,73 @@
+from nautobot.core.celery import register_jobs
+from nautobot.extras.choices import LogLevelChoices
+from nautobot.extras.datasources import ensure_git_repository, git_repository_dry_run, refresh_datasource_content
+from nautobot.extras.jobs import Job, ObjectVar
+from nautobot.extras.models import GitRepository
+
+name = "System Jobs"
+
+
+class GitRepositorySync(Job):
```

Comment:
Maybe set a `dryrun` BooleanVar here and merge the `GitRepositorySync` and `GitRepositoryDryRun` jobs?